dev-util/systemtap/systemtap-3.3.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit linux-info autotools eutils python-single-r1
```

Comment:
where is `eutils` being used?

Reply:
Search me? This is a straight-up copy from the 3.1-r1 ebuild.